pkg/operator/staticpod/installerpod/cmd.go

```diff
@@ -442,7 +463,13 @@ func (o *InstallOptions) Run(ctx context.Context) error {
 		return fmt.Errorf("failed to copy: %v", err)
 	}
 
-	recorder.Eventf("StaticPodInstallerCompleted", "Successfully installed revision %s", o.Revision)
+	waitStartedAt := time.Now()
+	if err := o.waitForPodMirror(ctx); err != nil {
+		recorder.Warningf("StaticPodInstallerMirrorFailed", "Failed to observe mirrored static pod %q revision %s in API server on node %q after %s of waiting: %v", o.StaticPodName, o.Revision, o.NodeName, time.Now().Sub(waitStartedAt), err)
```

Comment:
it won't put an operator into a degraded condition

Reply:
it could, we have event observer controller that can make operator degraded if we want to.

Reply:
would that be deterministic? I always thought events are not quartered to be delivered